docs/shadow-vs-active-enforcement/README.mdx

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 6
+title: 'Shadow vs active enforcement'
+---
+
+When installing Otterize, the intents operator can be [configured](/reference/configuration/intents-operator/helm-chart) to enforce access controls or not.
+
+By default, the intents operator enforces access controls by creating, updating and deleting network policies and Kafka ACLs, for example, but all enforcement can be disabled by setting `enableEnforcement` to be `false`, or more granularly setting `enableNetworkPolicyCreation` and/or `enableNetworkPolicyCreation` to be `false`.
+
+## Active enforcement
+
+When enforcement is active &mdash; when it is not disabled &mdash; the intents operator processes applied client intents files and turns each into access authorizations from the client to its called servers: if network policies are active (and supported by the CNI in the Kubernetes cluster), that ensuring there are network policies from that client to all its called servers, and removing any other previous network policies from that client; and similarly for Kafka ACLs to Kafka servers.
```

Comment:
Two suggestions:
1. Client intents files -> resources, since there's no concept of files when reasoning about an operator
2. removed the sentence 'removing any other previous network policies', since we don't modify network policies not owned (created) by the operator.
```suggestion
When enforcement is active &mdash; when it is not disabled &mdash; the intents operator processes applied client intents and turns each into access authorizations from the client to its called servers: if network policies are active (and supported by the CNI in the Kubernetes cluster), the operator creates network policies and labels pods in order to allow access from that client to all its called servers; and similarly for Kafka ACLs to Kafka servers.
```

Reply:
1. Agreed.
2. Yes, but we do remove network policies we put in based on previous extents, and that's important to prevent dangling permissions -- a core value for us. Can you find a way to pull that back in?

Reply:
How about this?
```suggestion
When enforcement is active &mdash; when it is not disabled &mdash; the intents operator processes applied client intents and turns each into access authorizations from the client to its called servers: if network policies are active (and supported by the CNI in the Kubernetes cluster), the operator labels pods and creates/updates network policies in order to allow access from that client to all its called servers; and similarly for Kafka ACLs to Kafka servers.
```

Reply:
I think it still misses making the point -- let me suggest something:

...the operator labels pods and creates/updates network policies to allow access from that client to all its called servers, as well as removing previously-granted access based on out of date intents;

Reply:
```suggestion
When enforcement is active &mdash; when it is not disabled &mdash; the intents operator processes applied client intents and turns each into access authorizations from the client to its called servers: if network policies are active (and supported by the CNI in the Kubernetes cluster), the operator labels pods and creates/updates network policies to allow access from that client to all its called servers, as well as removing previously-granted access based on out of date intents; and similarly for Kafka ACLs to Kafka servers.
```